scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/TaskData.java

```diff
@@ -101,7 +101,10 @@
 
 
 @Entity
-@NamedQueries({ @NamedQuery(name = "getFinishedTasksCount", query = "select count(*) from TaskData task where taskStatus in (:taskStatus) and task.jobData.removedTime = -1"),
+@NamedQueries({ @NamedQuery(name = "deleteTaskDataInBulk", query = "delete from TaskData where jobData.id in :jobIdList"),
+                @NamedQuery(name = "countTaskData", query = "select count (*) from TaskData"),
+                @NamedQuery(name = "countTaskDataNotFinished", query = "select count (*) from TaskData where taskStatus <> 11"),
```

Comment:
When we insert or change a task Status, isn't it possible that we move those numbers one up or down? Maybe you can infer the 11 from the actual task status?